src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -11,8 +11,11 @@ class Lambda {
   def process(inputStream: InputStream, outputStream: OutputStream) = {
     val apiUser = createUser(lambdaConfig.consignmentApiUser)
     sql"GRANT SELECT, INSERT, UPDATE ON ALL TABLES IN SCHEMA public TO $apiUser;".execute.apply()
+    sql"GRANT USAGE on consignment_sequence_id to $apiUser;".execute.apply()
```

Comment:
What will happen if this script is run before the DB migrations have been run for the first time?

Reply:
It'll fail but you kind of want it to because if you run this before the migrations are run then the users won't have access to any of the tables. It needs to be done after the initial migrations.

Reply:
That being said, we could get it to die slightly more gracefully by checking if the sequence is there first and then throwing an error if it's not. What do you think?

Reply:
So does script have to be run every time we do a DB migration which creates a table?

Reply:
I don't think that's the way to do it. This is to create the users, the grant statements for any new tables should probably go in the migration script when we create new tables.

Reply:
Yeah, that seems like something we'll need to figure out.

Either way it might be annoying if every time we create a new table, it works fine locally and then you run it on integration and it falls over because the permissions are missing. Is there any way round that?

Reply:
I've found [alter default privileges](https://www.postgresql.org/docs/9.4/sql-alterdefaultprivileges.html) which grants a user permissions for any future tables and I've added a line in to do this for the consignment_api_user. The migrations user doesn't need it because it will be creating the tables.
I've tested this by doing this for the bastion user using the admin user and then creating a new table and the bastion_user had permissions to select on the new table. I revoked the grant from the bastion user afterwards.